--- tpcd.h.orig	2014-05-16 02:56:06.748914416 +0900
+++ tpcd.h	2014-05-16 03:03:27.940901516 +0900
@@ -112,6 +112,15 @@
 #define SET_DBASE       ".LOGON %s\n"
 #endif
 
+#ifdef MYSQL
+#define GEN_QUERY_PLAN  "EXPLAIN"
+#define START_TRAN      "START TRANSACTION"
+#define END_TRAN        "COMMIT"
+#define SET_OUTPUT      "INTO OUTFILE"
+#define SET_ROWCOUNT    "LIMIT %d\n"
+#define SET_DBASE       "USE %s;\n"
+#endif
+
 #define MAX_VARS      8 /* max number of host vars in any query */
 #define QLEN_MAX   2048 /* max length of any query */
 #define QUERIES_PER_SET 22
